src/index.ts

```diff
@@ -27,7 +27,7 @@ const handleAuth = async (url: URL, env: Env) => {
 	const oauth2 = createOAuth(env);
 	const authorizationUri = oauth2.authorizeURL({
 		redirect_uri: `https://${url.hostname}/callback?provider=github`,
-		scope: 'public_repo,user',
+		scope: 'repo,user',
```

Comment:
If we're going to change the code and make it "private-first", should the readme instead comment on what you should do for the public repo case?

I think the readme change is fine if they're expected to make a change here for the private case on their own (ie: if the code stays as-is)

Reply:
Yes, pl ignore the code change. Just include the readme. Thanks a million.